contracts/Orchestrator.sol

```diff
@@ -123,12 +101,10 @@ contract Orchestrator is Ownable {
      * @param data The encoded data payload.
      * @return True on success
      */
-    function externalCall(address destination, bytes data)
-        internal
-        returns (bool)
-    {
+    function externalCall(address destination, bytes memory data) internal returns (bool) {
```

Comment:
should probably replace the assembly with a .call

Reply:
Makes sense 👍  We can address the Orchestrator changes separately ..
https://github.com/ampleforth/uFragments/issues/192